.github/test-a-feature.md

```diff
@@ -103,4 +103,69 @@ yarn test
 yarn test:watch
 ```
 
-[1]: https://github.com/stardust-ui/react/tree/master/test/specs/commonTests
\ No newline at end of file
+[1]: https://github.com/stardust-ui/react/tree/master/test/specs/commonTests
+
+## Behaviours tets
+
+Behavior unit tests are generated from the specification written in each behavior file.
+Each line under the "@specification" tag is taken and trying to match the regex experesion written in "testDefinitions.ts" file. 
+
+**Adding test(s)**
+
+What I need to do if I am adding ?:
+
+- New behavior file. In the file behavior-test.tsx add:
+  - your new added behavior into object: import {}
+  - to testHelper like: testHelper.addBehavior('yourNewBehaviorName', yourNewBehaviorImportedObject)
```

Comment:
maybe use `add it to the testHelper object like : testHelper.addBehavior('yourNewBehaviorName', yourNewBehaviorImportedObject)`